testgrid/config/config.yaml

```diff
@@ -2823,3 +2816,8 @@ dashboard_groups:
   - google-gke
   - google-gke-staging
   - google-gke-prod
+
+- name: sig-cluster-lifecycle-all
+  dashboard_names:
```

Comment:
http://k8s-testgrid.appspot.com/google-gke

for https://github.com/kubernetes/test-infra/pull/3122